fission-web-server/library/Fission/Web/Server.hs

```diff
@@ -41,16 +41,21 @@ import           Fission.Web.Server.Config.Types
 import           Fission.Web.Server.Types               as Fission
 
 import           Fission.Web.Server.Handler
+import qualified Fission.Web.Server.Handler.Relay       as Relay
+import           Fission.Web.Server.Handler.Relay.Types
+
 import qualified Fission.Web.Server.Host.Types          as Web
 import           Fission.Web.Server.IPFS.Linked
 import           Fission.Web.Server.MonadDB
 import           Fission.Web.Server.Reflective
+import           Fission.Web.Server.Relay.Store.Class
 
 import           Fission.Internal.Orphanage.OctetStream ()
 import           Fission.Internal.Orphanage.PlainText   ()
 
 -- | Top level web API type. Handled by 'server'.
-type API = Swagger.API :<|> Fission.API
+type API    = Swagger.API :<|> Fission.API :<|> LinkWS
+type LinkWS = "user" :> "link" :> RelayWS
```

Comment:
Unfortunately, the creators of the servant-ws library have client and server together, which brings in the XZ dependency that ideally we avoid in the CLI 😠  So putting this in the server directly.